backend/vaa-strapi/package.json

```diff
@@ -8,20 +8,11 @@
     "start": "strapi start",
     "build": "strapi build",
     "strapi": "strapi",
-    "lint": "eslint .",
```

Comment:
Anything to do with linting or formatting is removed from all of the workspaces and only handled by the scripts in the root package.json.